test/framework/test_composite.py

```diff
@@ -409,7 +409,12 @@ def test_composite_auto_save(self):
         expdata.service = service
         self.assertExperimentDone(expdata)
         expdata.auto_save = True
-        self.assertEqual(service.create_or_update_experiment.call_count, 3)
+        self.assertEqual(service.create_or_update_experiment.call_count, 2)
```

Comment:
Hmm it's weird. The API is called by this line, which is called by `ExperimentData.save_metadata`.
https://github.com/Qiskit-Extensions/qiskit-experiments/blob/aa3b54fee901858e650d7416972a26700ce09ca8/qiskit_experiments/framework/experiment_data.py#L1629-L1631

Can you show the callstack?

Reply:
I solved, with removing adding empty exp data and reconfigured test according to new imp

Reply:
'''console

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/home/musasina/Desktop/qiskit-experiments/test/framework/test_composite.py", line 437, in test_composite_auto_save
    self.assertEqual(service.create_or_update_experiment.call_count, 3)

      File "/usr/lib/python3.11/unittest/case.py", line 873, in assertEqual
    assertion_func(first, second, msg=msg)

      File "/usr/lib/python3.11/unittest/case.py", line 866, in _baseAssertEqual
    raise self.failureException(msg)

    AssertionError: 2 != 3


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    Experiment cannot be saved because no experiment service is available. An experiment service is available, for example, when using an IBM Quantum backend.

'''